### DISCUSSION TITLE
Unable to select fileField to upload a file

### DISCUSSION THREAD

**Discussion by amitsarkar007** [getgauge/taiko#1833] [Development]
Unable to execute line 23 on this code [Taiko-Jobserve.js](https://github.com/amitsarkar007/Taiko-Scripts/blob/0a9c7f19a9899125a6d45541ee3fc0213f4825f3/Taiko-JobServe.js#L23).

Executing the code to identify the element returns true but am still not able to attach the file. This used to work previously.

```
> fileField({id: 'LocalCVFile'}, {selectHiddenElements : true}).exists()
[PASS] Exists
true
```

Versions used on my machine
```
❯ taiko --version
Version: 1.1.4 (Chromium: 88.0.4314.0) RELEASE

❯ node --version
v14.15.4
```

**Comment by NivedhaSenthil** [comment]
Refer https://github.com/getgauge/taiko/issues/1771
